packages/block-editor/src/components/block-list/block.js

```diff
@@ -316,309 +292,295 @@ export class BlockListBlock extends Component {
 			case ENTER:
 				// Insert default block after current block if enter and event
 				// not already handled by descendant.
-				this.props.onInsertDefaultBlockAfter();
+				onInsertDefaultBlockAfter();
 				event.preventDefault();
 				break;
 
 			case BACKSPACE:
 			case DELETE:
 				// Remove block on backspace.
-				const { clientId, onRemove } = this.props;
 				onRemove( clientId );
 				event.preventDefault();
 				break;
 		}
-	}
+	};
 
-	onBlockError( error ) {
-		this.setState( { error } );
-	}
+	/**
+	 * Begins tracking cursor multi-selection when clicking down within block.
+	 *
+	 * @param {MouseEvent} event A mousedown event.
+	 *
+	 * @return {void}
```

Comment:
I think generally we're just not adding `@return` tag when the return value is `undefined`.